pkg/controllerutils/miscellaneous.go

```diff
@@ -27,6 +27,9 @@ import (
 	"k8s.io/utils/clock"
 )
 
+// DefaultReconciliationTimeout is the default timeout for the context of reconciliation functions.
+const DefaultReconciliationTimeout = 1 * time.Minute
```

Comment:
Do we still need this ?https://github.com/gardener/gardener/blob/2868508f994821c68fc37289aa97b49d43b856dc/pkg/gardenlet/apis/config/v1alpha1/types.go#L761-L762

Reply:
This doesn't seem to be addressed.

Reply:
I would also agree that the it's a bit clearer when 2 variables are used. It's just a coincidence that both are 1 minute.